unison-src/transcripts/lambdacase.output.md

```diff
@@ -141,7 +141,7 @@ blorf = cases
     ⍟ These new definitions are ok to `add`:
     
       structural type B
-      blah  : B -> B -> Text
+      blah  : B -> hod1jusqau2 -> Text
```

Comment:
this is weird, right?

Reply:
errr it's a universally quantified type that got named `hod1jusqau2`?

Reply:
This does seem odd. @tstat any idea what's going on here?

Reply:
Oh, wait, I get it. It's actually fine, it's just picking the synthetic variable name generated by `cases`

```
cases ...

converts to

synthetic19283 -> match synthetic19283 with ...
```

@tstat I'd update this example to force the second argument to be of type `B`, or else add a type signature.

Reply:
Oh whoops, I didn't notice this change. Thanks for catching it.

I added a commit that gives a type signature.